Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -513,37 +542,46 @@ instructionLabel(_call)
 
     # operation returns the entrypoint in r0 and the target instance in r1
     operationCall(macro() cCall3(_ipint_extern_prepare_call) end)
-    popQuad(sc1, t2)
+    loadp [sp], IPIntCallCallee
+    addq 16, sp
```

Comment:
This could just be popQuad again if you do add the  "post-indexing" instruction.